bindings/python/tests/test_offline.py

```diff
@@ -14,6 +14,7 @@
 client = Client()
 
 
+@pytest.mark.skip(reason="temporarily skipped")
```

Comment:
Can you review all the skips please? Honestly this doesn't really make sense, this is completely unrelated to blocks.

Reply:
okay, went through it again, this one indeed doesn't need to be ignored (anymore?)